spec/forms/pages/delete_condition_form_spec.rb

```diff
@@ -26,7 +26,7 @@
 
   describe "#delete" do
     context "when validation pass" do
-      let(:condition) { Condition.new id: 2, form_id: 1, page_id: 2, routing_page_id: 1, check_page_id: 1, answer_value: "Wales", goto_page_id: 3 }
+      let(:condition) { Condition.new id: 2, form_id: 1, page_id: 2, routing_page_id: 1, check_page_id: 1, answer_value: "Wales", goto_page_id: 3, skip_to_end: false }
```

Comment:
should we maybe use conditions factory?

Reply:
Hmm, when I use the conditions factory I get:
```
Failure/Error: result = record.destroy
ActiveResource::MissingPrefixParam:
form_id prefix_option is missing
```